contracts/ConsensusApp.sol

```diff
@@ -23,31 +23,25 @@ contract ConsensusApp is ForceMoveApp {
         ConsensusAppData memory appDataA = appData(a.appData);
         ConsensusAppData memory appDataB = appData(b.appData);
 
-        if (identical(a.outcome, b.outcome)) {
-            if (appDataB.furtherVotesRequired == numParticipants - 1) {
-                // propose
-                require(appDataA.furtherVotesRequired == 0, 'ConsensusApp: invalid propose, furtherVotesRequired must be transition from zero');
-                return true;
-            } else if (appDataB.furtherVotesRequired == 0) {
-                // veto or pass
-                require(appDataB.proposedOutcome.length == 0, 'ConsensusApp: invalid veto or invalid pass, proposedOutcome must transition to empty');
-                return true;
-            } else if (appDataB.furtherVotesRequired == appDataA.furtherVotesRequired - 1) {
-                // vote
-                require(appDataA.furtherVotesRequired > 1,'ConsensusApp: invalid vote, furtherVotesRequired must transition from at least 2');
+        if(appDataB.furtherVotesRequired == 0) { // final vote or veto/pass
+            require(appDataB.proposedOutcome.length == 0, 'ConsensusApp: proposedOutcome must be empty, if furtherVotesRequired = 0');
```

Comment:
Not clear to me whether this should be a requirement. It doesn't seem like anything goes wrong if you don't have this.